runtime/themes/everblush.toml

```diff
@@ -0,0 +1,132 @@
+# Author: Isotoxal <isotoxal@proton.me>
+
+"attribute" = { fg = "blue" }
+"boolean" = { fg = "cyan" }
+"character" = { fg = "blue" }
```

Comment:
Some of these scopes aren't used by Helix. These should be

```toml
"constant.builtin.boolean" = { fg = "cyan" }
"constant.character" = { fg = "blue" }
```

The captures in use are documented here: https://docs.helix-editor.com/master/themes.html#syntax-highlighting

Reply:
I am really sorry!!, I copied out a neovim theme's tree sitter and run a macro on it to convert it to toml. I didn't realize helix may not have some of the things used. Thank you!!!